src/cyGraph/CyGraph.tsx

```diff
@@ -1,14 +1,14 @@
-import { layoutDagre } from '../../utils';
 import Cytoscape, { ElementDefinition } from 'cytoscape';
 import { useEffect, useRef, useState } from 'react';
 import CytoscapeComponent from 'react-cytoscapejs';
-import { NodeType } from '../../models/nodeType';
-import { hasConnectorIri, imageHeightKey, imageKey, imageWidthKey } from '../../mapper/predicates';
-import { colorKey, labelKey, simpleSvgKey, labelIri } from '../../mapper/predicates';
-import { GraphProps } from '../../state/GraphStateProps';
-import { AbstractNode, GraphAssertion, GraphEdge, GraphPatch, GraphProperty } from '../../models/graphModel';
-import { NodeSymbol } from '../../symbol-api';
+import { NodeType } from '../core/types/nodeType';
+import { hasConnectorIri, imageHeightKey, imageKey, imageWidthKey } from '../core/mapper/predicates';
+import { colorKey, labelKey, simpleSvgKey, labelIri } from '../core/mapper/predicates';
+import { GraphProps } from '../core/state/GraphStateProps';
+import { AbstractNode, GraphAssertion, GraphEdge, GraphPatch, GraphProperty } from '../core/types/graphModel';
```

Comment:
Mb add `index.ts` for exports?

Reply:
We have an index.ts file in each folder, but don't export everything because that would expose everything for the user of our package. That way the end user only see the relevant types/functions/components. Makes sense?